src/main/kotlin/com/atlassian/performance/tools/report/api/judge/Verdict.kt

```diff
@@ -57,6 +77,7 @@ class Verdict internal constructor(
                 assertion = "$expectedReportCount reports were expected, but ${reports.size} reports were yielded",
                 testName = testName
             )
+
```

Comment:
This separates the `when` cases arbitrarily

Reply:
I didn't get this one. What do you mean?
Side note: taking a look at the original implementation of plus() operator intuitively it should also add failedReports

Reply:
`when` block is usually formatted in such way that it saves boilerplate compred to `if-elseif-...-else`. I believe it's its main purpose.

It also saves lines and code like:
```
when {
    conditionA() -> doA()
    conditionB() -> doB()
    conditionC() -> doC()
}
```
is more compact then
```
when {
    conditionA() -> doA()

    conditionB() -> doB()

    conditionC() -> doC()
}
```

The newlines are unnecessary

Reply:
Ah, <facepalm> I thought this comment is related to operator fun plus